pkg/apis/apps/doc.go

```diff
@@ -15,5 +15,6 @@ limitations under the License.
 */
 
 // +k8s:deepcopy-gen=package,register
+// +k8s:openapi-gen=true
```

Comment:
Yes, I had =package before but saw it redundant, the README file briefly cover this.